apps/extension/src/ui/domains/Portfolio/AssetsTable/AssetRowStakingReminderWrapper.tsx

```diff
@@ -0,0 +1,87 @@
+import { TALISMAN_WEB_APP_STAKING_URL } from "@core/constants"
+import { Balances } from "@core/domains/balances/types"
+import { isStakingSupportedChain } from "@core/domains/staking/helpers"
+import { ExternalLinkIcon, XIcon, ZapIcon } from "@talismn/icons"
+import { classNames } from "@talismn/util"
+import { useAnalytics } from "@ui/hooks/useAnalytics"
+import { useCallback } from "react"
+import { Trans, useTranslation } from "react-i18next"
+
+import { useStakingBanner } from "../../Staking/context"
+import { useTokenBalancesSummary } from "../useTokenBalancesSummary"
+
+export const AssetRowStakingReminderWrapper = ({
```

Comment:
Finding this wrapper/render pattern pretty confusing.
wdyt about moving the code from line 23 to 32 into a new hook that would be called in DashboardAssetRow, and conditionally render the staking reminder from there ? this would remove the need for the staking reminder to wrap the row